packages/@okta/vuepress-site/docs/guides/mfa/set-up-org/sms/enablemfa.md

```diff
@@ -0,0 +1,3 @@
+1. From the Admin Console, select **Security** from the main menu and then select **Multifactor**.
```

Comment:
....from the main menu, and then select Multifactor.

or

....from the main menu and then Multifactor.